### PR TITLE
Ignore false UI diffs in  va-banner and va-loading-indicator

### PR DESCRIPTION
**Pull request by ataker**
## Chromatic

https://dst2547-add-chromatic-ignores--65a6e2ed2314f7b8f98609d8.chromatic.com

### va-maintenance-banner
Diffs in va-maintenance-banner were caused by the spinning loading icon. Solution was to add a `isChromatic` check and if that is true add a class to the icon that prevents the animation from playing.

### va-loading-indicator
Diffs in va-loading-indicator were caused by the the dates changing in the banner. Solution was to hard code the dates. The side effect of this is that the error maintenance banner is only supposed to show if the maintenance is currently active, so I added an `isError` prop to force the error banner in storybook and chromatic. Test added for `isError` prop.

Closes [Add an ignore or threshold to false UI diffs in va-banner and va-loading-indicator #2547](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2547)

## QA Checklist
- [x] Component maintains 1:1 parity with design mocks
- [x] Text is consistent with what's been provided in the mocks
- [x] Component behaves as expected across breakpoints
- [ ] (N/A no accessibility changes made) Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] (N/A no design changes made)  Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [x] Tab order and focus state work as expected
- [ ]  (N/A no accessibility changes made) Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [x] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [x] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots

Screenshot 2024-03-07 at 17 07 46
Screenshot 2024-03-07 at 17 07 34

## Acceptance criteria
- [x] QA checklist has been completed
- [x] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by ataker** [comment]
It wasn't until writing up the PR that I realized we need to manually update the maintenance banner date in the future. The banners only show if the end time is in the future so having a hard coded date will break this once the date passes. Not sure if we want to push the date further out, create a reminder to update it, or find another solution. 

Seems like we would need to set the `diffThreshold` too high for the test to be useful at all and we can't ignore elements in the shadow DOM so I don't have another solution currently.

**Review by Andrew565 — APPROVED** [review]

**Comment by Andrew565** [comment]
I approved this, but I'd be more than happy to see the hardcoded date for the maintenance banner just pushed out to some absurd date like 2099 just so we don't have to worry about it in our life times.

**Comment by jamigibbs** [comment]
> I approved this, but I'd be more than happy to see the hardcoded date for the maintenance banner just pushed out to some absurd date like 2099 just so we don't have to worry about it in our life times.

I plan to live until 150! 😂

Does something like this work?

```
const year = new Date().getFullYear() + 50;
const maintenanceTime = new Date(year, 5, 21);
```

File: packages/web-components/src/components/va-checkbox/va-checkbox.scss [hunk 1]
```diff
@@ -18,6 +18,7 @@
 .usa-checkbox {
   max-width: 320px;
   background: transparent;
+  padding-bottom: 1px;
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-checkbox/va-checkbox.scss:21]: Can you add this to the Storybook example instead? Adding it here means we're diverging from USWDS styles.

Comment (ataker) [reply packages/web-components/src/components/va-checkbox/va-checkbox.scss]: This was just a test, and then it passed after removing this, so we should just keep an eye on [this test](https://www.chromatic.com/test?appId=65a6e2ed2314f7b8f98609d8&id=65ea3bc591ce6dd1397ab13f) in the future. Updating the storybook makes a lot more sense.

**Pull request merged**